assets/sass/mixins/_focus.scss

```diff
@@ -0,0 +1,14 @@
+@mixin focus-with-outline-only {
+	&.focus-visible{
+		outline: 3px solid $aplus-focus;
+	}
+}
+
+@mixin focus-with-background-and-color {
+	&.focus-visible{
+		background-color: $aplus-focus;
+    color: $aplus-foreground;
```

Comment:
mix of different indentation chars?

Reply:
I have no idea why this keeps happening, but thanks.

Reply:
Yeah. The `.editorconfig` seems to be used by your editor as it removes whitespace in end of lines.

Reply:
The weird indentation is still here.

Reply:
Fixed it manually. No idea why the editor is currently so hit-and-miss.